src/SDK/Trace/Sampler/TraceIdRatioBasedSampler.php

```diff
@@ -60,7 +60,7 @@ public function shouldSample(
         $traceIdCondition = $lowerOrderBytes < round($this->probability * $traceIdLimit);
         $decision = $traceIdCondition ? SamplingResult::RECORD_AND_SAMPLE : SamplingResult::DROP;
 
-        return new SamplingResult($decision, $attributes, $traceState);
+        return new SamplingResult($decision, [], $traceState);
```

Comment:
why not passing `$attributes` through?

Reply:
The returned attributes are additional attributes, ["A set of span Attributes that will also be added to the `Span`"](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/trace/sdk.md#shouldsample).